packages/airplane/internal/sleep/index.ts

```diff
@@ -0,0 +1,14 @@
+import ms, { StringValue } from "ms";
+
+import { getRuntime } from "../runtime";
+
+export const sleep = async (duration: StringValue) => {
+  if (!duration) {
+    throw new Error("sleep duration is required");
```

Comment:
This error message seems to make more sense up here?

```suggestion
    throw new Error("expected a duration for how long to sleep (e.g. '1h' to sleep for an hour)");
```